modules/base/rendering/renderableplane.cpp

```diff
@@ -41,17 +41,29 @@
 #include <variant>
 
 namespace {
+    enum RenderOption {
+        ViewDirection = 0,
+        PositionNormal,
+        FixedRotation
+    };
+
     enum BlendMode {
         Normal = 0,
         Additive
     };
 
-    constexpr openspace::properties::Property::PropertyInfo BillboardInfo = {
-        "Billboard",
-        "Billboard Mode",
-        "Specifies whether the plane should be a billboard, which means that it is "
-        "always facing the camera. If it is not, it can be oriented using other "
-        "transformations.",
+    constexpr openspace::properties::Property::PropertyInfo OrientationRenderOptionInfo =
+    {
+        "OrientationRenderOption",
+        "Orientation Render Option",
+        "Controls how the plane will be oriented. \"Camera View Direction\" rotates the "
+        "plane so that it is orthogonal to the viewing direction of the camera (useful "
+        "for planar displays), and \"Camera Position Normal\" rotates the plane towards "
+        "the position of the camera (useful for spherical displays, like dome theaters). "
+        "In both these cases the plane will be billboarded towards the camera but in a "
+        "slightly different way. In contrast, \"Fixed Rotation\" does not rotate the "
+        "plane at all based on the camera and should be used the plane should be "
+        "oriented in a fixed way.",
```

Comment:
Looks like this can maybe be used verbatim in part of the parameter docs for the billboard argument. Is that possible?

Reply:
I think its a thing that we have to address more holistically.  Since specifying a "default" value for a property description would be very misleading since users might interpret that as the value that it started with.

Post release, we'll have to think of a way to not duplicate the description between properties and parameter structs by still allowing for these subtle differences and then apply them to all applicable properties.